tests/UI/package.json

```diff
@@ -17,8 +17,8 @@
     "check:links": "ts-mocha tools/linkchecker.ts",
     "check:links:fast-fail": "ts-mocha --bail tools/linkchecker.ts",
     "// Specific test": "Script to run a specific test/folder",
-    "test:specific": "ts-mocha --file utils/setup.ts campaigns/$TEST_PATH",
-    "test:specific:fast-fail": "ts-mocha --bail --file utils/setup.ts campaigns/$TEST_PATH",
+    "test:specific": "ts-mocha $EXTRA_TEST_PARAMS --file utils/setup.ts campaigns/$TEST_PATH",
```

Comment:
Why you need `EXTRA_TEST_PARAMS` ?

Reply:
I need the extra params in this PR https://github.com/PrestaShop/ga.tests.ui.pr/pull/44
It allows to add the `--bail` to fast fail dynamically, the option is a checkbox in the run modal parameters, it will help debugging faster since I also added a new workflow to run one campaign only
Fast failure + single campaign should help us be more efficient when debugging UI tests

Reply:
I don't use for now in `test:specific:fast-fail` since the `--bail` is already there, but I though it could turn out useful in the future